model/src/main/kotlin/vulnerabilities/VulnerabilityReference.kt

```diff
@@ -30,6 +32,7 @@ import java.net.URI
  * with a list of references; each reference points to the source of the information and has some detailed information
  * provided by this source.
  */
+@JsonIgnoreProperties(value = ["severity_rating"], allowGetters = true)
```

Comment:
Should we also not serialize the property, because it is redundant? If so, probably it would be good to instead annotate the property with @JsonIgnore.

Reply:
> Should we also not serialize the property, because it is redundant?

That's more or less the discusion I had with @mnonnenmacher over [here](https://github.com/oss-review-toolkit/ort/pull/7762#discussion_r1376636535). Not serializing it would require to reintroduce the `EvaluatedVulnerabilityReference` class.

Reply:
My preference would (as well?) be to not have it serialized in OrtResult at all, because of the reasons mentioned there. However, that's out of scope of this PR.